androidshared/src/main/java/org/odk/collect/androidshared/ui/ObviousProgressBar.kt

```diff
@@ -3,23 +3,30 @@ package org.odk.collect.androidshared.ui
 import android.content.Context
 import android.os.Handler
 import android.util.AttributeSet
-import android.widget.ProgressBar
+import com.google.android.material.progressindicator.LinearProgressIndicator
 
 /**
  * A progress bar that shows for a minimum amount fo time so it's obvious to the user that
  * something has happened.
  */
-class ObviousProgressBar(context: Context, attrs: AttributeSet?) : ProgressBar(context, attrs) {
+class ObviousProgressBar(
+    context: Context,
+    attrs: AttributeSet?
+) : LinearProgressIndicator(context, attrs) {
```

Comment:
This looks a lot nicer! I think it needs to be set to be Indeterminate so it will animate properly. Right now it just shows as a blue bar.

Reply:
Right, fixed.